.github/actions/build-extra-tests/action.yml

```diff
@@ -0,0 +1,28 @@
+name: build-extra-tests
+description: 'Builds extra test required for some flows'
+
+inputs:
+  tools-version:
```

Comment:
Same as other action, this isn't needed since the caching is done outside of this action.